modules/troubleshooting-network-observability-must-gather.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+:_content-type: PROCEDURE
+[id="network-observability-must-gather_{context}"]
+= Using the must-gather tool
+You can use the must-gather tool to collect logs, metrics, and information about the Network Observability Operator resources as well as cluster-wide resources.
+
+.Procedure
+. Navigate to the directory where you want to store the must-gather data.
+. Run the following command to collect cluster-wide must gather resources:
++
+[source,terminal]
+----
+$ oc adm must-gather
+ --image-stream=openshift/must-gather
+ --image=quay.io/netobserv/must-gather
+----
+//Is there a file format that the data is saved as? For example, `must-gather/must-gather.tar.gz`.//
+. The data is saved as ?. You can upload this file to a support case on the Red Hat Customer Portal.
```

Comment:
I don't think we need to mention the data file format as this typical must-gather fmt can be `must-gather.tar.gz` or `must-gather.tar.xz` this isn't specific to NO